### PR TITLE
fix(graph): Use Vertex.pos as starting position

### PR DESCRIPTION
**Pull request by GordonSmith**
If the user has set a pos for a Vertex, use it as its starting position.

Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

### CODE REVIEW

File: packages/graph/src/Vertex.ts [hunk 1]
```diff
@@ -23,6 +23,7 @@ export class Vertex extends SVGWidget {
         this._icon = new Icon();
         this._textBox = new TextBox();
         this._annotationWidgets = {};
+        this.pos({ x: undefined, y: undefined });
```
Comment (jbrundage) [inline packages/graph/src/Vertex.ts]: Why is this necessary?

Comment (GordonSmith) [reply packages/graph/src/Vertex.ts]: As it defaults to {x: 0, y: 0} which may be a legitimate initial position...

Comment (GordonSmith) [reply packages/graph/src/Vertex.ts]: @jbrundage bump

**Comment by jbrundage** [comment]
@GordonSmith looks good
